tests/backup/backup_test.go

```diff
@@ -2893,7 +2893,7 @@ var _ = Describe("{ClusterBackupShareToggle}", func() {
 
 			//CreateSchedule backup
 			log.Infof("Backup schedule name - %v", backupName)
-			_, err = CreateScheduleBackup(backupName, SourceClusterName, backupLocationName, backupLocationUID, []string{bkpNamespaces[0]}, nil, orgID, "", "", "", "", periodicPolicyName, schPolicyUid, ctx)
+			err = CreateSchedule(backupName, SourceClusterName, backupLocationName, backupLocationUID, []string{bkpNamespaces[0]}, nil, orgID, "", "", "", "", periodicPolicyName, schPolicyUid, ctx)
```

Comment:
Why did you change the name CreateScheduleBackup to CreateSchedule? It is creating schedule backup only , right?
By "CreateSchedule" I am assuming you are creating the schedule policy

Reply:
As far as I can tell, if you want to do a scheduled backup, you need to create a schedule first. This is different from an immediate backup or making a schedule policy. Once you have a schedule, your backups will happen at set intervals. It can be confusing, but in my opinion, calling the function "CreateSchedule" would make more sense and be consistent with the UI.